src/forklift/engine.py

```diff
@@ -567,19 +567,31 @@ def _send_report_email(template, report_object, subject):
 
 def _clone_or_pull_repo(repo_name):
     '''repo_name: string - a github repository username/reponame format
+                  or an object with host and repo with the username/reponame syntax
```

Comment:
We should add examples of each of the accepted formats.

Reply:
to here or the readme?

Reply:
I added both...